### ISSUE TITLE
[FEATURE] - Add colab examples

### ISSUE THREAD

**Issue by galeone** [zurutech/ashpy#26]
**Describe the feature and the current behavior/state**

Right now there are no examples "click and go", to run on Google Colab. There are no notebooks that explain and show how to use AshPy and why this solution should be better or more useful than using the pure Keras API or other solutions.

**Will this change the current api? How?**

No, it doesn't.

**Who will benefit with this feature?**

Everyone. Since, OK examples in the documentation and in the `README` are cool, but having some colab notebook is more powerful and easy to share (moreover, it's more catchy and people love executing cells and learning)

**Comment by EmanueleGhelfi** [comment]
Agree. Let's remember Colab runs python 3.6 and AshPy is compatible with Python 3.7. We can write notebooks and wait for Colab to support python 3.7.

**Comment by galeone** [comment]
Yup - we can start creating some notebook, first run it on a Binder, so people can try ashpy - and when colab starts supporting Python 3.7 we'll switch

**Comment by EmanueleGhelfi** [comment]
Just to remember: check if it is possible to use this https://nbsphinx.readthedocs.io/en/0.5.1/ to include notebooks in the doc
